crossfire/__init__.py

```diff
@@ -66,28 +66,33 @@ def _flatten_df(row, column_name):
     )
 
 
+def is_empty(data):
+    if not HAS_PANDAS or (HAS_PANDAS and not isinstance(data, DataFrame)):
+        return not data
+    elif isinstance(data, DataFrame):
+        return data.empty
+    else:
+        return False
```

Comment:
Created the `is_empty` function to evaluate the `data` inpput and execute an early return, if necessary.